website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -310,59 +310,89 @@ Follow the steps below to install Appsmith on your Kubernetes cluster:
     ```bash
     helm repo update
     ```
+3. Generate the `values.yaml` config file
+    ```bash
+    helm show values appsmith-ee/appsmith --version 2.0.0 > values.yaml
+    ```
+4. Edit the `values.yaml` file and set the following parameters
+
+   ```yaml
+    applicationConfig:
+      APPSMITH_ENCRYPTION_PASSWORD: "<PASSWORD>"
+      APPSMITH_ENCRYPTION_SALT: "<SALT_PASSWORD>"
+      APPSMITH_LICENSE_KEY: "<LICENSE_KEY>"
+   ```
```

Comment:
The salt and password need to be filled in with randomly generated passwords. Can we add this detail. Currently, it doesn't indicate _what_ should be filled in for there variables.

Reply:
sure